### PR TITLE
Add line style in layer configuration

### LINKED ISSUE #431: Support dasharrays in line layers

### PR DESCRIPTION
**Pull request by sebbousquet**
Fix #431

### CODE REVIEW

File: src/app/services/main-form-manager/config-map-export-helper.ts [hunk 1]
```diff
@@ -153,6 +154,10 @@ export class ConfigMapExportHelper {
                 paint['line-opacity'] = this.getMapProperty(modeValues.styleStep.opacity, mode, colorService, taggableFields);
                 paint['line-color'] = color;
                 paint['line-width'] = this.getMapProperty(modeValues.styleStep.widthFg, mode, colorService, taggableFields);
+                const lineType = modeValues.styleStep.lineType;
+                if (lineType !== LINE_TYPE.solid) {
+                    paint['line-dasharray'] = LINE_TYPE_VALUES.get(lineType);
+                }
```
Comment (MohamedHamouGisaia) [inline src/app/services/main-form-manager/config-map-export-helper.ts:160]:
```suggestion
                } else {
                      delete paint['line-dasharray'];
                }
```

File: src/settings.yaml [hunk 2]
```diff
@@ -37,34 +37,34 @@ arlas_wui_url: "http://localhost:4200"
 links: [ { "icon": "lock", "name": "Google", "url": "http://www.google.com" }, { "icon": "public", "name": "Yahoo", "url": "http://www.yahoo.com" } ]
 basemaps: 
   - name: "Basic"
-    url: "https://api.maptiler.com/maps/basic/style.json?key=YOUR_KEY"
+    url: "https://api.maptiler.com/maps/basic/style.json?key=kO3nZIVLnPvIVn8AEnuk"
```
Comment (MohamedHamouGisaia) [inline src/settings.yaml:40]: forgotten keys

File: src/app/services/main-form-manager/config-map-export-helper.ts (hunk 1 above)
Comment (sebbousquet) [reply src/app/services/main-form-manager/config-map-export-helper.ts:162]: Fixed

File: src/settings.yaml (hunk 2 above)
Comment (sebbousquet) [reply src/settings.yaml]: Fixed

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**